### PR TITLE
Feature/model message like reply

### PR DESCRIPTION
**Pull request by ngerald51**
Please review the additions.

### CODE REVIEW

File: app/controllers/message_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class MessageController < ApplicationController
+
+	def create
```
Comment (nndd91) [inline app/controllers/message_controller.rb:3]: change to space

File: app/controllers/message_controller.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class MessageController < ApplicationController
+
+	def create
+		Following.create(from_user_id: params[:from_user_id],
```
Comment (nndd91) [inline app/controllers/message_controller.rb]: Remove logic

File: app/controllers/message_controller.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+    redirect_to message_path(params[:profile_id])
+  end
+
+  def index
+  	    @message = Message.find(params[:from_user_id])
+  end
+
+  def destroy
+  	Following.find(params[:id]).destroy
```
Comment (nndd91) [inline app/controllers/message_controller.rb]: remove logic

File: app/models/message.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class Message < ApplicationRecord
+  belongs_to :from_user_id, class_name: "User"
+  belongs_to :to_user_id, class_name: "User"
+
+  def self.check_exist?(user, to_user_id)
+    Following.exists?(from_user_id: user, to_user_id: to_user_id)
```
Comment (nndd91) [inline app/models/message.rb]: Message?

**Review by nndd91 — APPROVED** [review]
Merge after making the changes

File: app/controllers/message_controller.rb (hunk 1 above)
Comment (ngerald51) [reply app/controllers/message_controller.rb:3]: i have made changes to my sublime editor to have 2 tab spaces. 

thanks

File: app/controllers/message_controller.rb (hunk 2 above)
Comment (ngerald51) [reply app/controllers/message_controller.rb]: change Following to Message model. Thanks for the heads up

File: app/controllers/message_controller.rb (hunk 3 above)
Comment (ngerald51) [reply app/controllers/message_controller.rb]: change Following Model to Message model

File: app/models/message.rb (hunk 4 above)
Comment (ngerald51) [reply app/models/message.rb]: Change Following to Message Model. Thanks

**Pull request merged**